larq_compute_engine/mlir/transforms/quantize_patterns.td

```diff
@@ -69,3 +69,7 @@ def : Pat<(TFL_QuantizeOp
 def : Pat<(TFL_QuantizeOp (LQ_DequantizeOp:$output $input), $qtype),
           (LQ_DequantizeOp $input),
           [(HasOneUse $output)]>;
+
+def : Pat<(LQ_QuantizeOp (TFL_DequantizeOp:$output $input)),
+          (LQ_QuantizeOp $input),
+          [(HasOneUse $output)]>;
```

Comment:
This restores `v0.6.2` behaviour when converting `toy_int8_model` from the end2end unittests:

before | after (like v0.6.2)
---|---
<img width="207" alt="Screenshot 2022-04-06 at 13 00 42" src="https://user-images.githubusercontent.com/13285808/161970027-1d98a061-7197-4ad0-9bf9-055e1b81ec47.png"> | <img width="154" alt="Screenshot 2022-04-06 at 13 00 51" src="https://user-images.githubusercontent.com/13285808/161970033-10d9af03-9857-4fd7-ab2d-5a69752707b0.png">

I think this is due to the fact that fake quantizers might be handled slightly differently now. In practice this shouldn't matter much since it's not necessary/recommended to use a int8 quantizer in a feed forward network directly before a binarization.